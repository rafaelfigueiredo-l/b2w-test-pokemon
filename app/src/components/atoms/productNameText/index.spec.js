import ProductNameTextStrong from "./";

describe("<ProductNameTextStrong />", () => {
  const component = <ProductNameTextStrong>Texto</ProductNameTextStrong>;

  const mounted = mount(component);
  const wrapper = shallow(component);

  it("should mount correctly ProductNameTextStrong component", () => {
    expect(wrapper.find(".productNameTextStrong")).toHaveLength(1);
    expect(mounted).toMatchSnapshot();
  });

  it("should unmount correctly ProductNameTextStrong component", () => {
    expect(mounted.unmount()).toEqual({});
  });

  it("should have a children", () => {
    expect(wrapper.find(".productNameTextStrong").children()).toHaveLength(1);
  });
});
