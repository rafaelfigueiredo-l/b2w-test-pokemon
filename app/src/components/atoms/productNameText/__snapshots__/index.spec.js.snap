// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ProductNameTextStrong /> should mount correctly ProductNameTextStrong component 1`] = `
<styled.span
  className="productNameTextStrong"
>
  <span
    className="sc-AxjAm ffCWxz productNameTextStrong"
  >
    Texto
  </span>
</styled.span>
`;
